### PR TITLE
Get url data from `location` on HttpStatus.seeOther (303)

### PR DESCRIPTION
**Pull request by kamilszpila**

### CODE REVIEW

File: example/pubspec.lock [hunk 1]
```diff
@@ -14,7 +14,7 @@ packages:
       name: characters
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.0.0"
+    version: "1.1.0-nullsafety.3"
```
Comment (michalkarmelita) [inline example/pubspec.lock]: Why did you update this?

Comment (kamilszpila) [reply example/pubspec.lock]: I run `flutter pub get` & this updated these packages. Do you want to remove these updates from this PR?

Comment (michalkarmelita) [reply example/pubspec.lock]: We can update if it's just for a local fork. We shouldn't update if you're planning to raise a PR to the original repo.

Comment (kamilszpila) [reply example/pubspec.lock]: Ok. I reverted these changes 👍

**Review by michalkarmelita — APPROVED** [review]

**Pull request merged**